pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -40,9 +43,9 @@ func EvalAddSSTable(
 	ms := cArgs.Stats
 	mvccStartKey, mvccEndKey := storage.MVCCKey{Key: args.Key}, storage.MVCCKey{Key: args.EndKey}
 
-	// TODO(tschottdorf): restore the below in some form (gets in the way of testing).
```

Comment:
Do you happen to know what this was about?

Reply:
Oops, I'm not sure actually, I should have double-checked. @tbg is it safe to reintroduce a span here, wondering what testing it gets in the way of?

Reply:
Not sure any more, if the tests pass you should be good to go. Thanks for fixing my TODO :-)